packages/rpc-methods/src/restricted/dialog.ts

```diff
@@ -30,58 +30,23 @@ export enum DialogType {
   Prompt = 'Prompt',
 }
 
-const BaseFieldsStruct = object({
```

Comment:
I'm confused, I thought we wanted to allow for substituting the fields for the custom UI

Reply:
That's how I understood it yeah. Maybe that's not correct?